src/components/CellLineTable/DiseaseTableColumns.tsx

```diff
@@ -73,7 +73,7 @@ export const getDiseaseTableColumns = (
             dataIndex: "parentalLine",
             responsive: mdBreakpoint,
             render: (
-                parentalLine: UnpackedNormalCellLine,
```

Comment:
I noticed this while tracing parental line modal thumbnail, and it's been bugging me that `UnpackedNormalCellLine.parentalLine` is a string type. It seems like `ParentLine` already has all the properties the modal needs. Would it make sense to use `ParentLine` here or we're using `UnpackedNormalCellLine` for a specific reason? 

With either interface the modal works, but I agree we should add further checks to catch type definition mismatches, unused properties, etc.

Reply:
there is no data type in the database that's a parental line: it's a relational pointer to a normal cell line. so the data is a normal cell line. For all the normal cell lines, it's just a reference to WTC, so it doesn't need any other information. I'm not sure what this ParentLine type is, but it's probably a subset of an UnpackedNormalCellLine, which would make sense here if that's all that's being sent down